Apps/phironnet/ironnet_connector.py

```diff
@@ -353,6 +409,26 @@ def initialize(self):
         self._password = config.get('password')
         self._verify_server_cert = config.get('verify_server_cert')
 
+        # Alert Notification Configs
+        self._enable_alert_notifications = config.get('enable_alert_notifications')
+        alert_cats = config.get('alert_categories')
+        if alert_cats:
+            self._alert_categories = [str(cat).replace(" ", "_").upper() for cat in alert_cats.split(',')]
```

Comment:
We should also trim whitespace off each parsed alert category to minimize user error.